app/views/courses/subjects/_form_fields.html.erb

```diff
@@ -16,24 +16,22 @@
     </div>
 
     <% if course.level == "secondary" %>
-      <div style="margin-bottom: -20px">
-        <details class="govuk-details" role="group" data-qa="course__subordinate_subject_accordion">
-          <summary class="govuk-details__summary" role="button">
-            <span class="govuk-details__summary-text">
-              Add a second subject (optional)
-            </span>
-          </summary>
-          <div class="govuk-details__text" id="details-content" aria-hidden="true">
-            <p>Your first subject is the main one. It will appear first in the course title. It represents the bursary or scholarship available if applicable.</p>
+      <details class="govuk-details govuk-!-margin-top-6 govuk-!-margin-bottom-0" role="group" data-qa="course__subordinate_subject_details" <% if course.selected_subject_ids.second %>open<% end %>>
+        <summary class="govuk-details__summary" role="button">
+          <span class="govuk-details__summary-text">
+            Add a second subject (optional)
+          </span>
+        </summary>
+        <div class="govuk-details__text" id="details-content" aria-hidden="true">
```

Comment:
shouldn't the aria-hidden be set to "false" if the details element is open

Reply:
Good catch

Reply:
Going to remove this attribute as it isn't needed (it's not in the design system component)